chatkit-android/src/main/kotlin/com/pusher/chatkit/AndroidChatkitDependencies.kt

```diff
@@ -12,11 +12,11 @@ import okhttp3.OkHttpClient
  * [ChatkitDependencies] implementation for Android using [AndroidDependencies] to fulfil [PlatformDependencies].
  */
 data class AndroidChatkitDependencies @JvmOverloads constructor(
-        override val tokenProvider: TokenProvider,
-        override val okHttpClient: OkHttpClient? = null,
-        private val platformDependencies: PlatformDependencies = AndroidDependencies(),
-        override val logger: Logger = platformDependencies.logger,
-        private val context: Context? = null
+    override val tokenProvider: TokenProvider,
```

Comment:
that indentation here is also not standard in the IDE, we really need to check in the matching code style for it

Reply:
Yeah, I agree - i've responded in the other message thread with what I've done, but we should definitely make it as easy as possible for everyone else to integrate.